docs/reference/pvc-finalizer-downgrade-issue.md

```diff
@@ -0,0 +1,32 @@
+* TOC
+{:toc}
+
+---
+title: Kubernetes Downgrade issue from 1.10 to 1.9 due to PV/PVC Protection
+---
+
+## PV/PVC Protection in Kubernetes 1.10
+
+When enabled, [PV/PVC Protection](docs/admin/admission-controllers.md#storage-object-in-use-protection-beta) prevents PV/PVC from being removed when the finalizers are removed. 
+
+## Downgrading issue
+
+After downgrading from Kubernetes 1.10 to 1.9, PV/PVCs that are created in Kubernetes 1.10 with PVC Protection cannot be removed.
+
+## Workaround
+
+Currently PV/PVC finalizers have to be manually removed so PV/PVC can be removed after downgrading to Kubernetes 1.9.
```

Comment:
"Currently the `pv-protection` and `pvc-protection` finalizers must be removed manually before you downgrade so that PVs and PVCs can be removed after you downgrade to version 1.9. Here's what to do:"

yeah, I added "before you downgrade" because that's what it looks like from the following instructions, but it was not completely clear. Also adding another step to make when you do what even clearer.

Reply:
good to me